rts/ExternalAI/SSkirmishAICallbackImpl.cpp

```diff
@@ -67,7 +67,7 @@ static inline CAICheats* GetCheatCallBack(int skirmishAIId) { return &AI_LEGACY_
 
 
 static void CheckSkirmishAIId(int skirmishAIId, const char* caller) {
-	if (skirmishAIId >= 0 && skirmishAIId < MAX_AIS)
+	if (skirmishAIId >= 0 && skirmishAIId < SPECIAL_AI_ID)
```

Comment:
Uses related to normal IDs should stay using `MAX_AIS`

Reply:
changed to `MAX_AIS`